packages/feedback/src/core/integration.ts

```diff
@@ -64,7 +64,7 @@ export const buildFeedbackIntegration = ({
     autoInject = true,
     showEmail = true,
     showName = true,
-    showScreenshot = false,
+    showScreenshot = true,
```

Comment:
Just to clarify, with the current state of things, this will lazy load the screenshots integration unless it is already installed, both on NPM & CDN?

Reply:
Maybe @ryan953 can confirm, but yes I think so. Is that not something we want to do?

Reply:
Before #11791 we're shipping the sync strategy using the name `feedbackIntegration` in all places.

Long term, https://github.com/getsentry/sentry-javascript/pull/11791 sets things up. From that PR:
- `packages/browser/src/index.ts` is returning `feedbackSyncIntegration as feedbackIntegration`, so that should be the Sync strategy for NPM users, by default.
- `index.bundle.*` (there are two) returning `feedbackAsyncIntegration as feedbackIntegration,` which is the async strategy